ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -177,15 +183,54 @@ private PubsubMessageToTableRow(ValueProvider<String> tableSpecTemplate,
       throw new IllegalArgumentException("Resolved destination table does not exist: " + tableSpec);
     }
 
+    TableRow tableRow;
```

Comment:
```suggestion
    final TableRow tableRow;
```

I know Java supports checking that a `final` variable is set within all branches of an if statement; I assume we can get the same checking in a switch? Maybe that's not actually supported.

Reply:
looks like it works!